hoodie-client/src/main/java/com/uber/hoodie/HoodieWriteClient.java

```diff
@@ -733,6 +755,35 @@ private void rollback(List<String> commits) {
     }
     }
 
+  private void cleanTemporaryDataFiles() {
```

Comment:
javadocs

Reply:
Can we push this method into HoodieCopyOnWriteTable and share it between finalize & rollback paths for cleanup..  WriteClient should be oblivious of this as we discussed..